cloud/terraform/cmd/terraform-machine-controller/Dockerfile

```diff
@@ -24,6 +24,18 @@ RUN CGO_ENABLED=0 GOOS=linux go install -a -ldflags '-extldflags "-static"' sigs
 
 # Final container
 FROM alpine:3.7
-RUN apk --no-cache add --update ca-certificates bash openssh terraform
-RUN echo 'plugin_cache_dir = "$HOME/.terraform.d/plugin-cache"' >> ~/.terraformrc
+
+ENV TERRAFORM_VERSION=0.11.7
```

Comment:
The apk version was 0.11.0, and `terraform destroy` didn't have an auto-approve flag.